index.html

```diff
@@ -258,6 +278,80 @@ <h2>
           different values (or values in a different order).
         </dd>
       </dl>
+      <h3>Receiving new input values</h3>
+      <p>
+        When the system receives new input values from a gamepad, run the following steps:
+      </p>
+      <ol>
+        <li>Let |gamepad:Gamepad| be the {{Gamepad}} instance in |navigator:Navigator|.{{Navigator/[[gamepads]]}} representing the gamepad.
+        <li>Queue a task (where?) to <a>update gamepad state</a> for |gamepad|.
+      </ol>
+      <p>
+        To <dfn>update gamepad state</dfn> for |gamepad|, run the following steps:
+      </p>
+      <ol>
+        <li>Let |now| be the [=current high resolution time=].
+        <li>Set |gamepad|.{{Gamepad/timestamp}} to |now|.
+        <li>Run the steps for <a>map and normalize axes</a> with |gamepad|.
+        <li>Run the steps for <a>map and normalize buttons</a> with |gamepad|.
+        <li>If |navigator|.{{Navigator/[[hasGamepadGesture]]}} is `false` and |gamepad| <a>contains a gamepad user gesture</a>:
+          <ol>
+            <li>Set |navigator|.{{Navigator/[[hasGamepadGesture]]}} to `true`.
+            <li>For each |connectedGamepad:Gamepad| of |navigator|.{{Navigator/[[gamepads]]}}:
```

Comment:
Got motivated from our last tele conf and added nullable type support 😇

```suggestion
            <li>For each |connectedGamepad:Gamepad?| of |navigator|.{{Navigator/[[gamepads]]}}:
```

Reply:
Thanks!